_includes/markdown/Vue.md

```diff
@@ -0,0 +1,110 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript framework, Vue. Outside of The [Official Vue Documentation](https://vuejs.org/), you should find here personalized recommendations for using the framework based on the types of projects we typically see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [Official Vue Documentation](https://vuejs.org/v2/guide/).
+
+For general coding style recommendations please see the official [Vue style guide](https://vuejs.org/v2/style-guide/).
```

Comment:
Shouldn't we just follow our general JS best practices?

Reply:
This is more Vue-specific stuff, it doesn't interfere with our best practices and formatting

Reply:
Updated this a little to be more clear